knowledge-base/capturing-net-traffic.md

```diff
@@ -0,0 +1,131 @@
+- Toggle ON the [system capturing mode]({%slug capture-traffic-get-started%}#system-capturing).
+- Start your NET application and make the HTTPS request.
+
+Fiddler captures the HTTPS sessions and displays them in the Live Traffic grid.
+
+
+
+## Manual Configuration
+
+Sometimes, you might need to manually configure your NET applications to explicitly respect the Fiddler Everywhere proxy. The below sections provide details on how to configure your [NET Framework](#configuring-proxy-in-net-application) or [NETCore applications](#configuring-proxy-in-netcore-application) in case the automatic traffic capturing from [preconfigured terminal](#capturing-net-traffic-through-preconfigured-terminal) or via the [system proxy](#capturing-net-traffic-through-system-capturing) is not applicable.
+
+
+### Configuring Proxy in NET Application
+
+To temporarily connect a NET application to the Fiddler Everywhere local proxy, you can use the [**WebRequest.DefaultWebProxy**](https://learn.microsoft.com/en-us/dotnet/api/system.net.webrequest.defaultwebproxy?view=net-8.0) class to set a proxy:
+
+```c#
+WebRequest.DefaultWebProxy = new WebProxy("127.0.0.1", 8866);
+```
+
+Alternatively, you can specify a proxy inside the **yourappname.exe.config** file. If the NET application is running in your current user account, add the following content inside the configuration section:
+
+```xml
+<configuration>
+    <system.net>
+        <defaultProxy>
+            <proxy bypassonlocal="false" usesystemdefault="true" />
+        </defaultProxy>
+    </system.net>
+</configuration>
+```
+
+If the NET application runs in a different user account (for example, a Windows service), edit the **machine.config** file:
+```xml
+<!-- The following section is to force the use of Fiddler Everywhere for all applications, including those running in service accounts -->  
+<system.net>
+    <defaultProxy>
+        <proxy autoDetect="false" bypassonlocal="false" proxyaddress="http://127.0.0.1:8866" usesystemdefault="false" />
+    </defaultProxy>
+</system.net>
+```
+
+Or, manually specify the proxy on an individual WebRequest object:
+```c#
+objRequest = (HttpWebRequest)WebRequest.Create(url);
+objRequest.Proxy= new WebProxy("127.0.0.1", 8866);
+```
+
+#### Localhost Traffic
+
+NET might bypass the Fiddler Everywhere proxy for URLs containing **localhost** regardless of other settings. So, rather than using localhost, change your code to refer to a machine name, or use [Fiddler Everywhere alliases]({%slug how-to-capture-localhost-traffic%}#using-aliases).
+
+This URL **won't** appear in Fiddler:
+```
+http://localhost:4000/some-service/
+```
+
+These URLs **will** appear in Fiddler Everywhere: 
+```
+http://localhost.:4000/some-service/
+http://mymachine:4000/some-service/
+http://ipv4.fiddler:4000/some-service/
+```
+
+### Configuring Proxy in NETCore Application
+
+
+Setup the proxy via the **netsh** tool:
+```sh
+netsh winhttp set proxy 127.0.0.1:8866
```

Comment:
This is Windows only and .NET Core apps can be cross-platform. Perhaps you should also explain that for macOS/Linux they should use 

```
export http_proxy=http://127.0.0.1:8866
export https_proxy=http://127.0.0.1:8866
```

syntax or the start terminal option?